sci-calculators/bc-gh/bc-gh-6.2.2.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://git.yzena.com/gavin/bc"
+SRC_URI="https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="libedit readline"
+
+RDEPEND="
+	!readline? ( libedit? ( dev-libs/libedit:= ) )
+	readline? (
+		sys-libs/readline:=
+		sys-libs/ncurses:=
+	)"
+DEPEND="${RDEPEND}"
```

Whitespace is cheap, so use it to increase readability, especially when you have potentially confusing nested conditions. Move trailing `"` to the next line to reduce diff as things are added. Plus, in EAPI 7+ we generally put `DEPEND` first since often RDEP = DEP + something more, than the other way around (this wasn't the case before but the most common cases of DEP = RDEP + something more were replaced by BDEP).

```suggestion
DEPEND="
	!readline? (
		libedit? ( dev-libs/libedit:= )
	)
	readline? (
		sys-libs/readline:=
		sys-libs/ncurses:=
	)
"
DEPEND="
	${RDEPEND}
"
```

---

I fixed a typo in what you gave me, but otherwise did exactly what you asked.